docs/usage-as-operator.md

```diff
@@ -27,6 +27,7 @@ machineImages:
     regions:
     - name: eu-central-1
       ami: ami-034fd8c3f4026eb39
+      # architecture: amd64 # optional
```

Comment:
Maybe add a short sentence about this field after

> You have to map every version that you specify in `.spec.machineImages[].versions` here such that the AWS extension knows the AMI for every version you want to offer.

Reply:
PTAL c6870a2f6d57108c294fb0b0aec6a4ff2e075e0e.